modules/acm/src/main/java/edu/gemini/epics/acm/CaContinuousApplySenderImpl.java

```diff
@@ -0,0 +1,423 @@
+/*
+ * Copyright (c) 2016-2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+ * For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package edu.gemini.epics.acm;
+
+import edu.gemini.epics.EpicsReader;
+import edu.gemini.epics.EpicsWriter;
+import edu.gemini.epics.api.ChannelListener;
+import gov.aps.jca.CAException;
+import gov.aps.jca.TimeoutException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+public class CaContinuousApplySenderImpl<C extends Enum<C> & CarStateGeneric> implements ApplySenderWithResource {
```

Comment:
This special ApplySender handles the case of continuous Epics commands, where the corresponding CAR record indicates the activity of the command, and there is no other indication of just the start. The normal ApplySender would not work in these cases, because it would wait forever for the CAR record to go Idle

Reply:
Did we add tests for this case? With epics mocks?